cognite/powerops/resync/models/v2/dms/data_models/views/dayAheadBids-SHOPPriceScenario.view.yaml

```diff
@@ -0,0 +1,28 @@
+space: '{{dayahead_model_space}}'
+externalId: SHOPPriceScenario
+name: SHOPPriceScenario
+version: '{{dayaheadbid_SHOPPriceScenario_version}}'
+filter:
+  equals:
+    property:
+    - node
+    - type
+    value:
+      space: '{{type_space}}'
+      externalId: SHOPPriceScenario
+properties:
+  name:
+    container:
+      space: '{{dayahead_model_space}}'
+      externalId: SHOPPriceScenario
+      type: container
+    containerPropertyIdentifier: name
+    name: name
+    description: Name for the BidMethod
+  timeSeries:
```

Comment:
A few comments on this:
1. Should we call this property `price`, or `priceTimeSeries`, or is `timeSeries` specific enough because the type is called SHOP**Price**Scenario (I know the suggestion - timeSeries - comes from me, but...)
2. In addition to the time series external ID, scenarios are currently specified with transformations too (we often use the same time series for many scenarios, just with different transformations) - does it make sense to add transformations: [Transformation] here? I think this is what Julia has done in her new data models (and it would be good to align with those anyways).
3. Having a SHOP**Price**Scenario covers what we currently run with, but the current setup supports a more flexible scenario definition (we specify incremental mappings, and those happen to only have mutations of the price as of now, but they could just as well have other variations, like different inflow forecasts - would we like to keep that flexibility? Then we could just use the Scenario type from the CogShop data model instead (but then we would have to define scenarios per watercourse/ModelTemplate...)